### DISCUSSION TITLE
请问vendor_tools里的cmake-build可以用来编译处iOS的库吗

### DISCUSSION THREAD

**Discussion by daiziyuBigo** [Tencent/libpag#1571] [Q&A] [answered]
vendor_tools目录下有个cmake-build的可执行文件，用于编译构建windows的库。好像可以传参ios，但是编译总是出错，有大佬可以解答一下吗

**Comment by domchen** [comment]
错误日志是什么？

**Comment by kevingpqi123** [comment, marked as answer]
支持的，pag 的源码仓库中有 ios 的 demo 工程，ios 的库文件更推荐基于 demo 工程构建

**Comment by domchen** [comment]
vendor_tools里的工具除了编译windows平台可以在PC上用，编译其他平台都需要在Mac电脑上运行，包括iOS，Android和Web。你看看是不是在Windows电脑上去编译iOS了，这个是不可能实现的。
